File: ios/code/iOS基础/WKWebViewDemo/TestViewController.m
//
//  TestViewController.m
//  WKWebViewDemo
//
//  Created by WTW on 2020/3/25.
//  Copyright © 2020 WTW. All rights reserved.
//

#import "TestViewController.h"
#import "ViewController.h"

@interface TestViewController ()

@end

@implementation TestViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    self.view.backgroundColor = [UIColor whiteColor];
    
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    [self.navigationController pushViewController:[ViewController new] animated:YES];
}

@end

File: ios/code/Swift/ModalPresentationStyle/TestViewController.swift
//
//  TestViewController.swift
//  ModalPresentationStyle
//
//  Created by WTW on 2020/5/15.
//  Copyright © 2020 WTW. All rights reserved.
//

import UIKit

class TestViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        let view = UIView()
        
        view.frame = CGRect(x: 0, y: 100, width: UIScreen.main.bounds.size.width, height: UIScreen.main.bounds.size.height - 100)

        view.backgroundColor = .green
        
        self.view.addSubview(view)
        
        
        self.view.backgroundColor = .green
    }
    
    
    
    

   

}
